app/uk/gov/hmrc/fhregistrationfrontend/models/ModelEncryption.scala

```diff
@@ -48,4 +50,26 @@ object ModelEncryption {
     )
   }
 
+  def encryptSessionCache(
```

Comment:
See https://github.com/hmrc/ei-frontend/pull/220/files#diff-db93cae2aeca63146de522b94b717eaa25ceb76d32be1e9ea266d9502f716380R21

Needs to actually encrypt